_docs/powered-by-oak-price-apis.md

```diff
@@ -38,15 +38,20 @@ The following requirements apply for any Kusama parachain that wishes to support
     2b. [XCM converter implementation](https://github.com/OAK-Foundation/substrate-parachain-template/pull/14)
     
     2c. Get proxy accountId:
-```    
-curl --location --request POST 
-'http:*//rpc.turing-staging.oak.tech' \*
---header 'Content-Type: application/json' \
---data-raw '{"id":1, "jsonrpc":"2.0", "method": 
-"xcmpHandler_crossChainAccount", 
-"params": ["{{accountId32}}"]}' \
+
+```bash
+curl --location --request POST 'https://rpc.turing-staging.oak.tech' \
+  --header 'Content-Type: application/json' \
+  --data-raw '{
+   "id":1,
+   "jsonrpc":"2.0",
+   "method": "xcmpHandler_crossChainAccount",
+   "params": ["SS58_ACCOUNT_ID"]
+ }'
 ```
 
+Replace `SS58_ACCOUNT_ID` with the account you want to lookup.
```

Comment:
does it have to be in a specific prefix? e.g. `51`?

Reply:
This one does not.  The RPC will take in a wallet with any SS58 prefix and return the correct address with a substrate network prefix (`42`):

```
Querying with OAK network '6871Utgdq5ycWnTzm7VEwQfg4zM81JqYv8EDzKPu3fdzsXK8'
{"jsonrpc":"2.0","result":"5Egie3jrkENmFv5tUUcFCp72zz6xeL8JzJ8hjvKuuGcQySGD","id":1}
Querying with Substrate network '5E259xp1NU757FppcKa2YJGrFd4UWfAsDfF3TAcBsyHcLryL'
{"jsonrpc":"2.0","result":"5Egie3jrkENmFv5tUUcFCp72zz6xeL8JzJ8hjvKuuGcQySGD","id":1}
```

As far as I've tested, SS58 prefixed addresses can be used interchangeably cross chain with both the Polkadot Wallet App and the Polkadot.js API.